src/Sherpa/ArticleViewController.swift

```diff
@@ -73,17 +76,26 @@ internal class ArticleViewController: ListViewController {
 		
 		self.titleLabel.text = self.article.title
 		
-		self.bodyView.backgroundColor = UIColor.clear
-		self.bodyView.isEditable = false
-		self.bodyView.font = UIFont.preferredFont(forTextStyle: UIFontTextStyle.body)
-		self.bodyView.textColor = self.dataSource.document.articleTextColor
-		self.bodyView.tintColor = self.dataSource.document.tintColor
-		self.bodyView.translatesAutoresizingMaskIntoConstraints = false
-		self.bodyView.textContainer.lineFragmentPadding = 0
-		self.bodyView.textContainerInset = UIEdgeInsets.zero
-		self.contentView.addSubview(self.bodyView)
-		
-		self.bodyView.attributedText = self._applyAttributes(toString: self.article.body)
+        if self.article.html.count > 0 {
```

Comment:
The logic you've specified here should also be included in `ArticleTests`, specifically checking that a `html` value overrides any `body` value present.